### DISCUSSION TITLE
Cannot connect to SIP Server

### DISCUSSION THREAD

**Discussion by ansorikom** [tayler6000/pyVoIP#132] [Q&A]
I am tried to make IVR Bots Call, however there a problem when i am tried connect to SIP Server
i already test the SIP connection using X-Lite App (Windows) also using flashphoner.com and it working fine

Here the error i got 
```py
File "C:\Python39\lib\site-packages\pyVoIP\VoIP.py", line 656, in start
   self.sip.start()
  File "C:\Python39\lib\site-packages\pyVoIP\SIP.py", line 946, in start
    self.register()
  File "C:\Python39\lib\site-packages\pyVoIP\SIP.py", line 1702, in register
    raise TimeoutError("Registering on SIP Server timed out")
TimeoutError: Registering on SIP Server timed out
```

and this the class that i call 
```py
server = VoIPPhone(
    sip["SIPendpoint"], # prod.tncp.XXXXXXX.com
    sip["SIPport"], # 5070
    sip["username"], # 12XXXXX0747
    sip["password"],  # XXXXXXXXXX
    callCallback = answered,
    myIP = IPAddr,
)
```

**Comment by xyc0815** [comment]
Is the port 5070 open on your firewall? The normal SIP port is 5060.

**Comment by iHamzaShahid** [comment]
I am getting the same problem. Once I call the VoIPPhone class, it connects and registers itself to the server, but after some time it raises error "TimeoutError: Registering on SIP Server timed out". Firewall on both sides(server and client) are disabled. Any help or suggestion would be appreciated.

**Comment by xyc0815** [comment]
So do I understand it right. The register request itself runs and the client is connected to the server once and sometimes later you get a timeout?
Maybe there comes a keep alive request in between? To find out, you can add a debug output in the sip receiver.
